lib/SILOptimizer/Mandatory/TransferNonSendable.cpp

```diff
@@ -1392,7 +1392,11 @@ class BlockPartitionState {
   void print(llvm::raw_ostream &os) const {
     os << SEP_STR << "BlockPartitionState[reached=" << reached
        << ", needsUpdate=" << needsUpdate << "]\nid: ";
+#ifndef NDEBUG
     basicBlock->printID(os);
```

Comment:
There's a couple more `succ->printID(os)` and `pred->printID(os)` in lines 1412 and 1417. Don't those need to be protected for `NDEBUG` builds too?